docs/ensembl-help/getting-started/sequences.md

```diff
@@ -11,20 +11,22 @@ tags:
 status: draft
 ---
 
-# Get sequences
+# How to download sequences
 
-You can export sequences from the Genome browser and Entity viewer apps using the Sequence download tool:
+You can download gene, cDNA, CDS and peptide sequences from the Genome browser and Entity viewer apps.
 
-## Genome browser
+![](../../img/id-genome-browser.svg)
+
+In the Genome browser, sequence download is found in the Track drawer.
+
+![Genome browser drawer revealing download ](media/sequence-download-genome-browser.png)
 
 ![](../../img/id-genome-browser.svg)
-In the Genome browser, you can download gene and transcript sequences by clicking on the quick download link in the drawer or by clicking on the interactive transcript model and selecting the appropriate sequence to download.
 
 You can download sequences as a FASTA file which is suitable for using with sequence analysis tools.
 
-## Entity viewer
 ![](../../img/id-entity-viewer.svg)
-In the Entity viewer, you can download the sequence of your transcript of interest by clicking on the interactive transcript and selecting from the download options or by clicking on the download icon on the right hand side of the page.
 
+In the Entity viewer, you can download the sequence of your transcript of interest by selecting on the interactive transcript and selecting from the download options or by clicking on the download icon on the right hand side of the page.
```

Comment:
'selecting on the interactive transcript _image_'
This sentence might be a little misleading since the download icon is for *all* transcripts ?

Reply:
done